<!-- -->
<launch>
<!-- Camera1 -->
	<arg name="num_right" default="25000750" />
	<arg name="num_left" default="25001194" />
	<arg name="num_central" default="25000698" />
	<!-- Variable names camera1-->
	<arg name="camera_node1" default="left" />
	<arg name="calibration_file1" default="package://camera_array_arm/resources/new/camera_1.yaml" />
<!-- 		<arg name="calibration_file1" default="/home/paolo/telekyb_ws/src/cameraarrayproject/ccny_my_camera_drivers/resources/new/camera_2.xml" /> -->
	<!-- Variable names camera2-->
	<arg name="camera_node2" default="camera2" />
	<arg name="calibration_file2" default="package://camera_array_arm/resources/new/camera_2.yaml" />
	<!-- Variable names camera3-->
	<arg name="camera_node3" default="right" />
	<arg name="calibration_file3" default="package://camera_array_arm/resources/new/camera_3.yaml" />
	<!-- Variable names camera4-->
	<arg name="camera_node4" default="camera4" />
	<arg name="calibration_file4" default="" />
<!--	<node pkg="odroid_usbio_driver" type="usbio" name="usbio" output="screen" >
	</node>-->
	<!--Node to visualize the camera 2 and 3  -->
	<node pkg="camera_array_arm" type="ndvi_computer" name="ndvi_computer" output="screen" >
		<param name="namespace_camL" type="string" value="cameraL/" />
		<param name="namespace_camC" type="string" value="cameraC/" />
		<param name="namespace_camR" type="string" value="cameraR/" />
		<param name="cameraL/serial_number" type="string" value="$(arg num_left)" />
		<param name="cameraC/serial_number" type="string" value="$(arg num_central)" />
		<param name="cameraR/serial_number" type="string" value="$(arg num_right)" />
		<param name="cameraL/calibration_url_" type="string" value="$(arg calibration_file1)" />
		<param name="cameraC/calibration_url_" type="string" value="$(arg calibration_file2)" />
		<param name="cameraR/calibration_url_" type="string" value="$(arg calibration_file3)" />
		<param name="cameraL/show_frames" type="bool" value="false" />
		<param name="cameraC/show_frames" type="bool" value="true" />
		<param name="cameraR/show_frames" type="bool" value="false" />
	<!-- parameters common to all cameras -->
<!-- 		<param name="calibration_file_$(arg num_left)" type="string" value="$(arg calibration_file3)" /> -->
		<param name="width" type="int" value="752" />
		<param name="height" type="int" value="480" />
		<param name="auto_gain" type="bool" value="true" />
		<param name="use_color" type="bool" value="true" /> 
		<param name="frame_id" type="string" value="bluefox" />
		<param name="expose_us" type="int" value="6000" /> 
		<param name="fps" type="int" value="10" />
<!-- left camera    <param name="num_camera" type="string" value="25001194" /> -->
<!--     <param name="calibration_file" type="string" value="$(arg calibration_file1)" /> -->
<!-- central camera    <param name="num_camera" type="string" value="25000698" /> -->
<!--     <param name="calibration_file" type="string" value="$(arg calibration_file2)" /> -->
	</node>
</launch>
